### ISSUE TITLE
"Long descriptions" - Complex Images

### ISSUE THREAD

**Issue by MakotoUeki** [w3c/wai-tutorials#630]
Suggestion from the Alt Text subgroup of the Silver Task force. I'm a lead of the subgroup.

Long descriptions - Complex Images
https://www.w3.org/WAI/tutorials/images/complex/#long-descriptions

The first paragraph reads:
"There are situations where the composition of an image is important and needs to be provided in the long description. For example, the sequence of colors used and the relative heights of the columns in a bar chart may be relevant information about the structure of the chart, in addition to the actual values and trends that it depicts."

The Alt Text subgroup would like to suggest the following wording to explain more specifically:
"Every image has a context (relationship to surrounding content). The context of an image determines its purpose. The purpose of the image determines the relevant information in the image. Generally, the long description should describe what the image represents, rather than how it appears. But, in some cases, the appearance of the image is relevant. For example, if the purpose of a flowchart is to demonstrate a process or relationship, then the long description should describe the process or relationship. But, if the purpose of the flowchart is to provide an example of flowchart design, then the long description should describe the boxes, arrows, and labels used to form the flowchart."

The Alt Text subgroup is reviewing the Tutorials to create several Method documents for WCAG 3. We reconfirmed that the Tutorial documents are very useful and helpful. Thank you all very much for your continuous meaningful work!

**Comment by yatil** [comment]
Thanks, @MakotoUeki, for the suggestion. I wonder if that is a bit too voluminous (and WCAG-sounding, to be honest) for this description. I think the first half about context applies to all images and should be put somewhere general.
